flutter/lib/src/sentry_flutter.dart

```diff
@@ -48,6 +48,9 @@ mixin SentryFlutter {
   }) async {
     final flutterOptions = SentryFlutterOptions();
 
+    // ignore: invalid_use_of_internal_member
+    sentrySetupStartTime = flutterOptions.clock();
```

Comment:
```suggestion
    sentrySetupStartTime ??= flutterOptions.clock();
```
so if the user initializes the SDK multiple times it's not an issue